### PR TITLE
Create tree view for navigation drawer

### PR DESCRIPTION
**Pull request by GrantPerkins**

### CODE REVIEW

**Comment by GrantPerkins** [comment]
Shoot, based in wrong branch. one sec

File: packages/client/src/components/navigationDrawer/LinkList.tsx [hunk 1]
```diff
@@ -1,38 +1,80 @@
...
-import { OverridableComponent } from "@material-ui/core/OverridableComponent";
-import { SvgIconTypeMap } from "@material-ui/core/SvgIcon/SvgIcon";
-import { makeStyles } from "@material-ui/core/styles";
 import { ReactElement } from "react";
+import { ChevronRight, ExpandMore, Folder, Info, PermMedia } from "@material-ui/icons";
+import gql from "graphql-tag";
+import { TreeView } from "@material-ui/lab";
+import { useQuery } from "@apollo/client";
+import { TreeGetProjectList } from "./__generated__/TreeGetProjectList";
+import { TreeGetDatasetList } from "./__generated__/TreeGetDatasetList";
```
Comment (phonymacoroni) [inline packages/client/src/components/navigationDrawer/LinkList.tsx:14]: Hey friend, wrong import

**Review by phonymacoroni — DISMISSED** [review]
Change to import from not generated things

File: packages/client/src/components/navigationDrawer/LinkList.tsx (hunk 1 above)
Comment (GrantPerkins) [reply packages/client/src/components/navigationDrawer/LinkList.tsx:14]: See other files. It is normal to import from generated files. Specifically, https://github.com/wpilibsuite/Axon/blob/0371b0ac03e6ec72b58696120665eba5feed0cd3/packages/client/src/components/datasets/Datasets.tsx#L17

**Pull request closed without merging**